spec/components/sidebar_controls_component_spec.rb

```diff
@@ -33,6 +33,7 @@
       let(:catkey) { nil }
 
       it 'creates a hash with the needed button info for an admin' do
+        allow(doc).to receive(:item?).and_return(true)
```

Comment:
this shouldn't be necessary as it already returns true due to https://github.com/sul-dlss/argo/pull/2733/files#L29